n_helpers.c

```diff
@@ -326,49 +210,143 @@ uint32_t NoteBinaryMaxDecodedLength(uint32_t bufferSize)
+{
+    // Validate parameter(s)
+    if (!len) {
+        const char *err = ERRSTR("len cannot be NULL", c_bad);
+        NOTE_C_LOG_ERROR(err);
+        return err;
+    }
+
+    // Issue a "card.binary" request.
+    J *rsp = NoteRequestResponse(NoteNewRequest("card.binary"));
+    if (!rsp) {
+        const char *err = ERRSTR("unable to issue binary request", c_err);
+        NOTE_C_LOG_ERROR(err);
+        return err;
+    }
+
+    // Ensure the transaction doesn't return an error and confirm the binary
+    // feature is available.
+    if (NoteResponseError(rsp)) {
+        NOTE_C_LOG_ERROR(JGetString(rsp, "err"));
+        JDelete(rsp);
+        const char *err = ERRSTR("unexpected error received during handshake", c_err);
+        NOTE_C_LOG_ERROR(err);
+        return err;
+    }
+
+    // Examine "length" from the response to evaluate the length of the decoded
+    // data residing on the Notecard.
+    *len = JGetInt(rsp, "length");
+    JDelete(rsp);
+
+    return NULL;
+}
+
+//**************************************************************************/
+/*!
+  @brief  Get the required buffer length to receive the entire binary object
+          stored on the Notecard.
+
+  @param  len [out] The length required to hold the entire contents of the
+           Notecard's binary data store. If there's no data stored on the
+           Notecard, then `len` will return 0.
+
+  @returns An error string on error and NULL on success.
+ */
+/**************************************************************************/
+const char * NoteBinaryStoreEncodedLength(uint32_t *len)
+{
+    // Validate parameter(s)
+    if (!len) {
+        const char *err = ERRSTR("size cannot be NULL", c_err);
+        NOTE_C_LOG_ERROR(err);
+        return err;
+    }
+
+    // Issue a "card.binary" request.
+    J *rsp = NoteRequestResponse(NoteNewRequest("card.binary"));
+    if (!rsp) {
+        const char *err = ERRSTR("unable to issue binary request", c_err);
+        NOTE_C_LOG_ERROR(err);
+        return err;
+    }
+
+    // Ensure the transaction doesn't return an error and confirm the binary
+    // feature is available.
+    if (NoteResponseError(rsp)) {
+        NOTE_C_LOG_ERROR(JGetString(rsp, "err"));
+        JDelete(rsp);
+        const char *err = ERRSTR("unexpected error received during handshake", c_bad);
+        NOTE_C_LOG_ERROR(err);
+        return err;
+    }
+
+    // Examine "cobs" from the response to evaluate the space required to hold
+    // the COBS-encoded data to be received from the Notecard.
+    long int cobs = JGetInt(rsp, "cobs");
+    JDelete(rsp);
+    *len = cobs;
+
+    return NULL;
+}
+
 //**************************************************************************/
 /*!
   @brief  Receive a large binary object from the Notecard's binary buffer
 
-  @param  buffer A buffer to hold the binary object
-  @param  bufLen The total length of the provided buffer
+  @param  buffer        A buffer to hold the binary range
```

Comment:
I'm good with either convention, but just noting that we aren't aligning the param descriptions for some of the other APIs in this PR. Also, prefer consistency with ending descriptions with a period.